Index.html

```diff
@@ -27,5 +27,113 @@ <h4 class="connect">LET'S CONNECT</h4>
       <li><img src="Assests/Vectorangerllist.svg"></li>
     </ul>
   </section>
+  <section class="worksSection">
+    <div class="space"></div>
+    <div class="firstCard">
```

Comment:
[OPTIONAL]
- [ ] your project image doesnt span across the card, please see the figma template and work on this change 
- figma design
![i-n-f](https://user-images.githubusercontent.com/89055502/235336604-c1e4a582-b239-4be7-aaeb-70ee05ea1c4a.png)
- your design
![i-n-s](https://user-images.githubusercontent.com/89055502/235336613-9c7a4ffe-edce-4cc7-8f6d-e55c3165b55b.png)